boards/microchip/mec_assy6941/board.yml

```diff
@@ -0,0 +1,17 @@
+boards:
+  - name: mec_assy6941_mec1743qlj
+    vendor: microchip
+    socs:
+      - name: mec1743_qlj
+  - name: mec_assy6941_mec1743qsz
+    vendor: microchip
+    socs:
+      - name: mec1743_qsz
+  - name: mec_assy6941_mec1753qlj
+    vendor: microchip
+    socs:
+      - name: mec1753_qlj
+  - name: mec_assy6941_mec1753qsz
+    vendor: microchip
+    socs:
+      - name: mec1753_qsz
```

Comment:
whilst you can do this, I don't see why you wouldn't just add 1 board with 4 socs then build for e.g. `mec_assy6941/mec1743_qsz`

Reply:
Thanks for the hint.  Changed it to be one board with choice of 1 of 4 socs.